combat-trainer.lic

```diff
@@ -384,6 +388,14 @@ class LootProcess
     return if Time.now - @loot_timer < @loot_delay
 
     game_state.mob_died = true
+    if (Time.now - @last_rites_timer > 600) && @last_rites
```

Comment:
Since there is some roundtime involved, do you want to retreat if there are still mobs alive in the room?